docs/source/stats.rst

```diff
@@ -28,18 +28,15 @@ Residual Diagnostics and Specification Tests
 .. module:: statsmodels.stats.stattools
    :synopsis: Statistical methods and tests that do not fit into other categories
 
+Autocorrelation
```

Comment:
I attempt to classify the methods, but I am uncertain about their academic accuracy.